src/pages/Version/Version.test.tsx

```diff
@@ -18,8 +18,7 @@ describe('test Version page', () => {
   jest.setTimeout(40000000);
   beforeAll(() => {
     // FIXME: a better way to mock this
```

Comment:
We can declare this VERDACCIO_API_URL as a global variable in the jest config if we want this to always be 'http://test'

Reply:
Agree 👍